### PR TITLE
CDD-2674: Updated sbt and scala

### PR DESCRIPTION
**Pull request by rashadmughal**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -88,7 +89,7 @@ lazy val commonSettings: Seq[Setting[_]] =
     defaultSettings() ++
     gitStampSettings
 
-lazy val playPublishingSettings: Seq[sbt.Setting[_]] = sbtrelease.ReleasePlugin.releaseSettings ++
+lazy val playPublishingSettings: Seq[sbt.Setting[_]] = Seq(credentials += SbtCredentials) ++
```
Comment (glidester) [inline build.sbt]: Duplication of 'Seq(credentials += SbtCredentials)'

File: LICENSE [hunk 2]
```diff
@@ -198,4 +199,4 @@
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
-   limitations under the License.
+   limitations under the License.
```
Comment (glidester) [inline LICENSE]: What has changed?

Comment (rashadmughal) [reply LICENSE]: It's part of precheck, the license must be like this otherwise the build fails

File: build.sbt (hunk 1 above)
Comment (rashadmughal) [reply build.sbt]: fixed

**Pull request merged**